src/cordova/android/README.md

```diff
@@ -12,7 +12,7 @@ Additional requirements for Android:
   - Optional for building, but useful for development
 - [Latest Android Sdk Commandline Tools](https://developer.android.com/studio/command-line)
   - Place it at `$HOME/Android/Sdk/cmdline-tools/latest`
-- Android SDK 30 (with build-tools) via commandline `$HOME/Android/Sdk/cmdline-tools/latest/bin/sdkmanager "platforms;android-30" "build-tools;30.0.3"`
+- Android SDK 31 (with build-tools) via commandline `$HOME/Android/Sdk/cmdline-tools/latest/bin/sdkmanager "platforms;android-31" "build-tools;31.0.0"`
```

Comment:
We need to update the tools we are using in the CI here: https://github.com/Jigsaw-Code/outline-client/blob/5bb2640a4ce761102ed64e658e46f7c15d9d8ac8/tools/build/setup_linux_android.sh#L35